<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="My sample Suite" duration-ms="54473" started-at="2018-05-04T18:40:47Z" finished-at="2018-05-04T18:41:41Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="54473" started-at="2018-05-04T18:40:47Z" finished-at="2018-05-04T18:41:41Z">
      <class name="com.crm.qa.test.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="setUp" is-config="true" duration-ms="38727" started-at="2018-05-04T18:40:47Z" finished-at="2018-05-04T18:41:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginPageTitle()[pri:1, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="loginPageTitle" duration-ms="67" started-at="2018-05-04T18:41:25Z" finished-at="2018-05-04T18:41:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [#1 Free CRM for Any Business] but found [Free CRM software in the cloud powers sales and customer service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [#1 Free CRM for Any Business] but found [Free CRM software in the cloud powers sales and customer service]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.crm.qa.test.LoginPageTest.loginPageTitle(LoginPageTest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="tearDown" is-config="true" duration-ms="860" started-at="2018-05-04T18:41:26Z" finished-at="2018-05-04T18:41:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="setUp" is-config="true" duration-ms="11065" started-at="2018-05-04T18:41:26Z" finished-at="2018-05-04T18:41:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:2, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="loginTest" duration-ms="2628" started-at="2018-05-04T18:41:37Z" finished-at="2018-05-04T18:41:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.test.LoginPageTest@a3e297]" name="tearDown" is-config="true" duration-ms="978" started-at="2018-05-04T18:41:40Z" finished-at="2018-05-04T18:41:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.crm.qa.test.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- My sample Suite -->
</testng-results>
